src/utils.ts

```diff
@@ -9,36 +9,30 @@
 
 'use strict';
 
-const utils = module.exports = {};
-
-function repeatChar(char, len) {
+export function repeatChar(char: string, len: number): string {
   if (len <= 0) {
     return '';
   }
   return new Array(len + 1).join(char);
 }
-utils.repeatChar = repeatChar;
 
-function padLeft(str, len) {
+export function padLeft(str: string, len: number): string {
   return (repeatChar(' ', len) + str).slice(-len);
 }
-utils.padLeft = padLeft;
 
-function padRight(str, len) {
+export function padRight(str: string, len: number): string {
   return (str + repeatChar(' ', len)).slice(0, len);
 }
-utils.padRight = padRight;
 
-function hexStr(num, len) {
+export function hexStr(num: number, len: number): string {
   return (repeatChar('0', len) + num.toString(16)).slice(-len);
 }
-utils.hexStr = hexStr;
 
-function alignCenter(str, len) {
+export function alignCenter(str: string, len: number): string {
   if (str.length >= len) {
     return str.slice(0, len);
   }
-  const leftSpace = parseInt((len - str.length) / 2);
+  const leftSpace = parseInt("" + (len - str.length) / 2);
```

Comment:
`parseInt` expects a string.
Maybe `parseInt` could be omitted?

Reply:
Yeah... no idea what the intent of this was. Seems reasonable to just remove `parseInt()`.